article/app/controllers/LiveBlogController.scala

```diff
@@ -287,7 +295,14 @@ class LiveBlogController(
   )(implicit request: RequestHeader): Result = {
     val pageType: PageType = PageType(blog, request, context)
     val model =
-      DotcomRenderingDataModel.forLiveblog(blog, blocks, request, pageType, filterKeyEvents, request.forceLive)
+      DotcomRenderingDataModel.forLiveblog(
```

Comment:
I can see down in the PR files, this method has one more extra parameters for topic list but here it doesn't. Also I can't find the method itself that has updated here. Maybe as a result of merge conflict this happened?